### PR TITLE
TypeScript support

### PR DESCRIPTION
**Pull request by keksipurkki**
Here's a minor enhancement for all the TypeScript users.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		"test": "xo && ava"
 	},
 	"files": [
+                "index.d.ts",
```
Comment (Qix-) [inline package.json]: Tabs, please.

Comment (keksipurkki) [reply package.json]: Got it.

**Comment by SamVerschueren** [comment]
Sorry, it's better to publish them to DefinitelyTyped. For more information, see https://github.com/sindresorhus/ama/issues/439

**Comment by sindresorhus** [comment]
I now accept TypeScript definitions if anyone looking is interested in submitting it here.

See: https://github.com/sindresorhus/ama/issues/439#issuecomment-414212229

Please follow this guide: https://github.com/sindresorhus/typescript-definition-style-guide

**Pull request closed without merging**